### PR TITLE
Various fixes in machines UI

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

File: pkg/machines/components/storagePools/storagePoolOverviewTab.jsx [hunk 1]
```diff
@@ -18,48 +18,30 @@
...
+            </React.Fragment> }
+
+            <label className='control-label' htmlFor={`${idPrefix}-persistent`}> Persistent: </label>
+            <div id={`${idPrefix}-persistent`}>{storagePool.persistent ? 'yes' : 'no' }</div>
+
+            <label className='control-label' htmlFor={`${idPrefix}-autostart`}> Autostart: </label>
+            <div id={`${idPrefix}-autostart`}>{storagePool.autostart ? 'yes' : 'no' }</div>
+
+            <label className='control-label' htmlFor={`${idPrefix}-type`}> Type: </label>
+            <div id={`${idPrefix}-type`}>{storagePool.type }</div>
```
Comment (martinpitt) [inline pkg/machines/components/storagePools/storagePoolOverviewTab.jsx]: I wonder why eslint doesn't complain about the imbalanced space here.

File: pkg/machines/components/vmOverviewTab.jsx [hunk 2]
```diff
@@ -23,7 +23,7 @@ import cockpit from 'cockpit';
 
 const _ = cockpit.gettext;
 
-const Items = ({ items, idPrefix, divider, colClass }) => {
+const Items = ({ items, idPrefix }) => {
```
Comment (martinpitt) [inline pkg/machines/components/vmOverviewTab.jsx]: not important, but the commit message says "These very properties" but it means "were"

File: pkg/machines/components/storagePools/storagePoolOverviewTab.jsx [hunk 3]
```diff
@@ -20,26 +20,29 @@ import React from 'react';
...
         <div className='ct-form-layout'>
             { storagePool.path && <React.Fragment>
-                <label className='control-label' htmlFor={`${idPrefix}-path`}> Path: </label>
+                <label className='control-label' htmlFor={`${idPrefix}-path`}> {_("Path")} </label>
                 <div id={`${idPrefix}-path`}>{storagePool.path}</div>
             </React.Fragment> }
 
-            <label className='control-label' htmlFor={`${idPrefix}-persistent`}> Persistent: </label>
+            <label className='control-label' htmlFor={`${idPrefix}-persistent`}> {_("Persistent")} </label>
             <div id={`${idPrefix}-persistent`}>{storagePool.persistent ? 'yes' : 'no' }</div>
```
Comment (martinpitt) [inline pkg/machines/components/storagePools/storagePoolOverviewTab.jsx]: The yes/no need to be translated, too.

File: pkg/machines/components/storagePools/storagePoolOverviewTab.jsx (hunk 1 above)
Comment (KKoukiou) [reply pkg/machines/components/storagePools/storagePoolOverviewTab.jsx]: No idea. Fixed.

File: pkg/machines/components/storagePools/storagePoolOverviewTab.jsx (hunk 3 above)
Comment (KKoukiou) [reply pkg/machines/components/storagePools/storagePoolOverviewTab.jsx]: Done

File: pkg/machines/components/vmOverviewTab.jsx (hunk 2 above)
Comment (KKoukiou) [reply pkg/machines/components/vmOverviewTab.jsx]: Fixed

**Pull request merged**